app/controllers/api/projects.php

```diff
@@ -133,6 +141,14 @@
             throw new Exception(Exception::PROJECT_RESERVED_PROJECT, "'console' is a reserved project.");
         }
 
+        // One in 20 projects use shared tables
+        $shareTables = !\mt_rand(0, 19);
```

Comment:
Im not sure how this works? This will generate a random number between 0 to 19. 
Which means there is a 1 / 20 chance of it returning 0, which means it will create 19 projects on shared tables, contrary to what is mentioned in the comment ?

Reply:
The `!` is inverting it so only when it returns 0 will it create the project on shared tables